plugins/catalog/src/components/CatalogTable/OffsetPaginatedCatalogTable.tsx

```diff
@@ -69,7 +69,7 @@ export function OffsetPaginatedCatalogTable(
       }}
       totalCount={totalItems}
       localization={{ pagination: { labelDisplayedRows: '' } }}
-      isLoading={isLoading}
```

Comment:
note this didn't have title (or subtitle) at all, which is why the example app (`yarn dev`) catalog table was missing the title.